container/Dockerfile.none

```diff
@@ -46,7 +46,7 @@ ARG CARGO_BUILD_JOBS
 
 ENV CARGO_TARGET_DIR=/workspace/target
 
-RUN cargo build --release --locked --features mistralrs,sglang,vllm,python && \
```

Comment:
no mistral?

Reply:
I'm guessing it's a default feature, but just double checking

Reply:
```
[features]
default = ["mistralrs", "llamacpp"]
```
Oh yeah!

I figured out that llamacpp gets statically linked, so we can include it by default effectively for free.

On top of that vllm and sglang are now sub-process, so they don't need a build flag. My next task is removing the build-your-own-engine ("python" feature flag). That will leave us with only cuda/metal/vulkan as feature flags.

Reply:
I put the mistralrs flag back, might as well be explicit. I also removed llamacpp from default features as that should be it's own commit.

Reply:
Sounds good to me - thanks 🚀